requirements/app/cloud.txt

```diff
@@ -4,4 +4,3 @@ redis>=4.0.1, <=4.2.4
 docker>=5.0.0, <=5.0.3
 # setuptools==59.5.0
 s3fs>=2022.5.0, <2022.8.3
-aiohttp>=3.8.0, <=3.8.3
```

Comment:
There are `aiohttp` and `lightning_api_access` in components.txt, too, just in case you've missed.

Reply:
@hhsecond said it is fine :)